hazelcast/src/main/java/com/hazelcast/query/impl/QueryableEntry.java

```diff
@@ -63,13 +63,19 @@ public Object getAttributeValue(String attributeName) throws QueryException {
     }
 
     public abstract K getKey();
+
```

Comment:
In some classes you remove the spacing, in some you add the spacing :)

Reply:
My idea puts spacing for methods when I auto-format.
But I admit that I have deleted some spacing between class fields.